.circleci/config.yml

```diff
@@ -1,8 +1,23 @@
+---
 version: 2.1
 
+parameters:
```

Comment:
The idea is that in each branch we would change this parameters right?

Reply:
exactly!